BREAKING.md

```diff
@@ -168,6 +170,31 @@ The `experimentalTransitionShadow` config option has been removed. The transitio
 
 The `Config.set()` method has been removed. See https://ionicframework.com/docs/angular/config for examples on how to set config globally, per-component, and per-platform.
 
+### React
```

Comment:
Should we also note that `setupConfig` is no longer exported for Angular and Vue?